src/output-handler.cc

```diff
@@ -40,10 +42,18 @@ namespace kcov
 			if (!conf.keyAsInt("cobertura-only"))
 			{
 				(void)unlink(readableName.c_str());
-				if (symlink(m_outDirectory.c_str(), readableName.c_str()) < 0)
-				{
-					kcov_debug(INFO_MSG, "Can't symlink readable name\n");
-				}
+
+                char buf[MAXPATHLEN + 1];
+                char *out_path = realpath(m_outDirectory.c_str(), buf);
```

Comment:
Nice! An improvement though: there is a get_real_path in utils.hh, which outputs a std::string

Reply:
done.